Documentation/helm-operator.md

```diff
@@ -131,7 +131,8 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.registrar.image`              | Kubernetes CSI registrar image.                                                                                                | `k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1` |
 | `csi.resizer.image`                | Kubernetes CSI resizer image.                                                                                                  | `k8s.gcr.io/sig-storage/csi-resizer:v1.0.0`               |
 | `csi.provisioner.image`            | Kubernetes CSI provisioner image.                                                                                              | `k8s.gcr.io/sig-storage/csi-provisioner:v2.0.0`           |
-| `csi.snapshotter.image`            | Kubernetes CSI snapshotter image.                                                                                              | `k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.0`           |
+| `csi.snapshotter.image`(K8s<=v1.16)| Kubernetes CSI alpha snapshotter image.                                                                                        | `k8s.gcr.io/sig-storage/csi-snapshotter:v1.2.2`           |
+| `csi.snapshotter.image`(K8s>=v1.17)| Kubernetes CSI snapshotter image.                                                                                              | `k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0`           |
```

Comment:
If these values are unset, these can be selected based on the kubernetes version in the helm chart yaml files.